app/assets/stylesheets/brands/_show.scss

```
@@ -0,0 +1,75 @@
+.itemIndex{
+  background-color: white;
+  width: 100vw;
+  &__title{
+    padding-right: 200px;
+    padding-left: 200px;
+    position: relative;
+    border-top: 1px solid #eee;
+    background: #f5f5f5;
+     -webkit-box-shadow: 0 0px 0px 0 rgba(0,0,0,0.16);
+    box-shadow: 0 3px 3px 0 rgba(0,0,0,0.16);
+    background: #fff;
+    z-index: 1;
+    height: 50px;
+    line-height: 50px;
+    font-family: 'Source Sans Pro', Helvetica , Arial, 'æ¸¸ã‚´ã‚·ãƒƒã‚¯ä½“', 'YuGothic', 'ãƒ¡ã‚¤ãƒªã‚ª', 'Meiryo', sans-serif;
+    font-size: 14px;
+    color: #333;
+  }
+  &__subtitle{
+    padding-right: 200px;
+    padding-left: 200px;
+    padding-top: 40px;
```

ショートハンドで書く方がすっきりして個人的には好きです
```suggestion
    padding: 40px 200px 0 200px;
```

---

修正しました